admin/src/components/Wysiwyg/wrapper.js

```diff
@@ -2,13 +2,16 @@ import styled from 'styled-components';
 import { Box } from "@strapi/design-system/Box";
 
 const Wrapper = styled(Box)`
+    padding-bottom: 2.8rem;
     @media (min-width: 651px) {
         .codex-editor--narrow .codex-editor__redactor {
             margin-right: 0;
         }
     }    
     .codex-editor {
-        padding: 1rem;
+        padding: 16px;
+        font-size: 1rem;
+        font-family: "Lato";
```

Comment:
Why is the font Lato, and not the standard fonts of the operating system, or fonts from Strapi?

For example, something like this: `"--apple-system", BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;`

Reply:
Good question. It was actually Lato before I even touched it. I can remove, and it can default to Strapi standard.